ibis/formats/pandas.py

```diff
@@ -263,7 +263,7 @@ def convert_Decimal(cls, s, dtype, pandas_type):
             scale=dtype.scale,
             strict=False,
         )
-        return s.map(func, na_action="ignore")
+        return s.map(func)
```

Comment:
Don't we want to leave NA value alone?

Reply:
This is to convert floating nan values to `Decimal("Nan")`

Reply:
Ah, okay! Carry on, then.